.github/workflows/action_lint.yml

```diff
@@ -1,15 +1,7 @@
 ---
 name: Action Lint
 
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    branches:
-      - main
-    paths:
-      - '.github/**'
+on: [push]
```

Comment:
I would prefer to keep the old way.
I don't see any benefits of running a CI outside of a PR (except for dev|main), this way we prevent useless builds, save penguins and money.